packages/suite/src/storage/CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Storage changelog
 
+## 18
```

Comment:
is there any motivation to bump by 2 versions instead of one?

Reply:
Yes, in the second one we alter already created table.

![image](https://user-images.githubusercontent.com/3112191/94923378-0d64ba80-04bc-11eb-9371-95e1c06622b4.png)

Reply:
i don't understand. why descriptor can't be added to the version 17 right away?
both versions (17 and 18) are created in this branch and wasn't released yet so there is no one who could possibly have version 17 => there is no one who need a migration to 18

Reply:
Okay, removed.

Reply:
It has been already used by our internal testers and we wanted to be nice to them. However, now we know that migration version is a precious resource and will act accordingly in the future :-)